v.11.x native image build fails

Latest release fails during build:

```
Fatal error: com.oracle.svm.core.util.VMError$HostedError: com.oracle.svm.core.util.VMError$HostedError: New Method or Constructor found as reachable after static analysis: public java.lang.String org.asamk.signal.manager.storage.groups.LegacyGroupStore$Storage$GroupV1$JsonRecipientAddress.number()
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.util.VMError.shouldNotReachHere(VMError.java:72)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:682)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.run(NativeImageGenerator.java:521)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.buildImage(NativeImageGeneratorRunner.java:407)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.build(NativeImageGeneratorRunner.java:585)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.main(NativeImageGeneratorRunner.java:128)
Caused by: com.oracle.svm.core.util.VMError$HostedError: New Method or Constructor found as reachable after static analysis: public java.lang.String org.asamk.signal.manager.storage.groups.LegacyGroupStore$Storage$GroupV1$JsonRecipientAddress.number()
------------------------------------------------------------------------------------------------------------------------
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.util.VMError.shouldNotReachHere(VMError.java:68)
	at org.graalvm.nativeimage.builder/com.oracle.svm.reflect.hosted.ReflectionFeature.getOrCreateAccessor(ReflectionFeature.java:109)
	at org.graalvm.nativeimage.builder/com.oracle.svm.reflect.hosted.ExecutableAccessorComputer.compute(ExecutableAccessorComputer.java:50)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.substitute.ComputedValueField.computeValue(ComputedValueField.java:384)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.substitute.ComputedValueField.readValue(ComputedValueField.java:353)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.meta.ReadableJavaField.readFieldValue(ReadableJavaField.java:38)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.ameta.AnalysisConstantReflectionProvider.readValue(AnalysisConstantReflectionProvider.java:102)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.meta.HostedField.readValue(HostedField.java:161)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.image.NativeImageHeap.addObjectToImageHeap(NativeImageHeap.java:439)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.image.NativeImageHeap.addObject(NativeImageHeap.java:295)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.image.NativeImageHeap.processAddObjectWorklist(NativeImageHeap.java:598)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.image.NativeImageHeap.addTrailingObjects(NativeImageHeap.java:198)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:664)
	... 4 more
```

Just tried to add the field to the reflect config, but more kept on popping up one after another. Then i started the JRE build with graalvm reflect agent (`JAVA_TOOL_OPTIONS="-agentpath:/tmp/graalvm-ce-java17-22.2.0/lib/libnative-image-agent.so=config-merge-dir=/tmp/signal-cli/graalvm-config-dir/"`) but unluckily, it did not merged anything.

--- a/graalvm-config-dir/reflect-config.json
+++ b/graalvm-config-dir/reflect-config.json
@@ -1055,21 +1055,38 @@
   "allDeclaredFields":true,
   "queryAllDeclaredMethods":true,
   "queryAllDeclaredConstructors":true,
-  "methods":[{"name":"<init>","parameterTypes":["java.util.List"] }]
+  "methods":[
+    {"name":"<init>","parameterTypes":["java.util.List"] },
+    {"name":"groups","parameterTypes":[] }
+  ]
 },
 {
   "name":"org.asamk.signal.manager.storage.groups.LegacyGroupStore$Storage$GroupV1",
   "allDeclaredFields":true,
   "queryAllDeclaredMethods":true,
   "queryAllDeclaredConstructors":true,
-  "methods":[{"name":"<init>","parameterTypes":["java.lang.String","java.lang.String","java.lang.String","java.lang.String","int","boolean","boolean","java.util.List"] }]
+  "methods":[
+    {"name":"<init>","parameterTypes":["java.lang.String","java.lang.String","java.lang.String","java.lang.String","int","boolean","boolean","java.util.List"] },
+    {"name":"groupId","parameterTypes":[] },
+    {"name":"expectedV2Id","parameterTypes":[] },
+    {"name":"name","parameterTypes":[] },
+    {"name":"color","parameterTypes":[] },
+    {"name":"messageExpirationTime","parameterTypes":[] },
+    {"name":"blocked","parameterTypes":[] },
+    {"name":"archived","parameterTypes":[] },
+    {"name":"members","parameterTypes":[] }
+  ]
 },
 {
   "name":"org.asamk.signal.manager.storage.groups.LegacyGroupStore$Storage$GroupV1$JsonRecipientAddress",
   "allDeclaredFields":true,
   "queryAllDeclaredMethods":true,
   "queryAllDeclaredConstructors":true,
-  "methods":[{"name":"<init>","parameterTypes":["java.lang.String","java.lang.String"] }]
+  "methods":[
+    {"name":"<init>","parameterTypes":["java.lang.String","java.lang.String"] },
+    {"name":"uuid","parameterTypes":[] },
+    {"name":"number","parameterTypes":[] }
+  ]
 },
 {
   "name":"org.asamk.signal.manager.storage.groups.LegacyGroupStore$Storage$GroupV1$MembersDeserializer",
@@ -1080,14 +1097,26 @@
   "allDeclaredFields":true,
   "queryAllDeclaredMethods":true,
   "queryAllDeclaredConstructors":true,
-  "methods":[{"name":"<init>","parameterTypes":["java.lang.String","java.lang.String","java.lang.String","boolean","boolean"] }]
+  "methods":[
+    {"name":"<init>","parameterTypes":["java.lang.String","java.lang.String","java.lang.String","boolean","boolean"] },
+    {"name":"groupId","parameterTypes":[] },
+    {"name":"masterKey","parameterTypes":[] },
+    {"name":"distributionId","parameterTypes":[] },
+    {"name":"blocked","parameterTypes":[] },
+    {"name":"permissionDenied","parameterTypes":[] }
+  ]
 },
 {
   "name":"org.asamk.signal.manager.storage.identities.LegacyIdentityKeyStore$IdentityStorage",
   "allDeclaredFields":true,
   "queryAllDeclaredMethods":true,
   "queryAllDeclaredConstructors":true,
-  "methods":[{"name":"<init>","parameterTypes":["java.lang.String","int","long"] }]
+  "methods":[
+    {"name":"<init>","parameterTypes":["java.lang.String","int","long"] },
+    {"name":"addedTimestamp","parameterTypes":[] },
+    {"name":"trustLevel","parameterTypes":[] },
+    {"name":"identityKey","parameterTypes":[] }
+  ]
 },
 {
   "name":"org.asamk.signal.manager.storage.profiles.LegacyProfileStore",
@@ -1150,28 +1179,60 @@
   "allDeclaredFields":true,
   "queryAllDeclaredMethods":true,
   "queryAllDeclaredConstructors":true,
-  "methods":[{"name":"<init>","parameterTypes":["java.util.List","long"] }]
+  "methods":[
+    {"name":"<init>","parameterTypes":["java.util.List","long"] },
+    {"name":"lastId","parameterTypes":[] },
+    {"name":"recipients","parameterTypes":[] }
+    ]
 },
 {
   "name":"org.asamk.signal.manager.storage.recipients.LegacyRecipientStore2$Storage$Recipient",
   "allDeclaredFields":true,
   "queryAllDeclaredMethods":true,
   "queryAllDeclaredConstructors":true,
-  "methods":[{"name":"<init>","parameterTypes":["long","java.lang.String","java.lang.String","java.lang.String","java.lang.String","org.asamk.signal.manager.storage.recipients.LegacyRecipientStore2$Storage$Recipient$Contact","org.asamk.signal.manager.storage.recipients.LegacyRecipientStore2$Storage$Recipient$Profile"] }]
+  "methods":[
+    {"name":"<init>","parameterTypes":["long","java.lang.String","java.lang.String","java.lang.String","java.lang.String","org.asamk.signal.manager.storage.recipients.LegacyRecipientStore2$Storage$Recipient$Contact","org.asamk.signal.manager.storage.recipients.LegacyRecipientStore2$Storage$Recipient$Profile"] },
+    {"name":"id","parameterTypes":[] },
+    {"name":"number","parameterTypes":[] },
+    {"name":"uuid","parameterTypes":[] },
+    {"name":"profileKey","parameterTypes":[] },
+    {"name":"expiringProfileKeyCredential","parameterTypes":[] },
+    {"name":"contact","parameterTypes":[] },
+    {"name":"profile","parameterTypes":[] }
+  ]
 },
 {
   "name":"org.asamk.signal.manager.storage.recipients.LegacyRecipientStore2$Storage$Recipient$Contact",
   "allDeclaredFields":true,
   "queryAllDeclaredMethods":true,
   "queryAllDeclaredConstructors":true,
-  "methods":[{"name":"<init>","parameterTypes":["java.lang.String","java.lang.String","int","boolean","boolean","boolean"] }]
+  "methods":[
+    {"name":"<init>","parameterTypes":["java.lang.String","java.lang.String","int","boolean","boolean","boolean"] },
+    {"name":"name","parameterTypes":[] },
+    {"name":"color","parameterTypes":[] },
+    {"name":"messageExpirationTime","parameterTypes":[] },
+    {"name":"blocked","parameterTypes":[] },
+    {"name":"archived","parameterTypes":[] },
+    {"name":"profileSharingEnabled","parameterTypes":[] }
+  ]
 },
 {
   "name":"org.asamk.signal.manager.storage.recipients.LegacyRecipientStore2$Storage$Recipient$Profile",
   "allDeclaredFields":true,
   "queryAllDeclaredMethods":true,
   "queryAllDeclaredConstructors":true,
-  "methods":[{"name":"<init>","parameterTypes":["long","java.lang.String","java.lang.String","java.lang.String","java.lang.String","java.lang.String","java.lang.String","java.lang.String","java.util.Set"] }]
+  "methods":[
+    {"name":"<init>","parameterTypes":["long","java.lang.String","java.lang.String","java.lang.String","java.lang.String","java.lang.String","java.lang.String","java.lang.String","java.util.Set"] },
+    {"name":"lastUpdateTimestamp","parameterTypes":[] },
+    {"name":"givenName","parameterTypes":[] },
+    {"name":"familyName","parameterTypes":[] },
+    {"name":"about","parameterTypes":[] },
+    {"name":"aboutEmoji","parameterTypes":[] },
+    {"name":"avatarUrlPath","parameterTypes":[] },
+    {"name":"mobileCoinAddress","parameterTypes":[] },
+    {"name":"unidentifiedAccessMode","parameterTypes":[] },
+    {"name":"capabilities","parameterTypes":[] }
+  ]
 },
 {
   "name":"org.asamk.signal.manager.storage.recipients.RecipientAddress",
@@ -1188,14 +1249,22 @@
   "allDeclaredFields":true,
   "queryAllDeclaredMethods":true,
   "queryAllDeclaredConstructors":true,
-  "methods":[{"name":"<init>","parameterTypes":["java.util.List"] }]
+  "methods":[
+    {"name":"<init>","parameterTypes":["java.util.List"] },
+    {"name":"sharedSenderKeys","parameterTypes":[] }
+  ]
 },
 {
   "name":"org.asamk.signal.manager.storage.senderKeys.LegacySenderKeySharedStore$Storage$SharedSenderKey",
   "allDeclaredFields":true,
   "queryAllDeclaredMethods":true,
   "queryAllDeclaredConstructors":true,
-  "methods":[{"name":"<init>","parameterTypes":["long","int","java.lang.String"] }]
+  "methods":[
+    {"name":"<init>","parameterTypes":["long","int","java.lang.String"] },
+    {"name":"recipientId","parameterTypes":[] },
+    {"name":"deviceId","parameterTypes":[] },
+    {"name":"distributionId","parameterTypes":[] }
+  ]
 },
 {
   "name":"org.asamk.signal.manager.storage.stickerPacks.JsonStickerPack",
@@ -1228,14 +1297,22 @@
   "allDeclaredFields":true,
   "queryAllDeclaredMethods":true,
   "queryAllDeclaredConstructors":true,
-  "methods":[{"name":"<init>","parameterTypes":["java.util.List"] }]
+  "methods":[
+    {"name":"<init>","parameterTypes":["java.util.List"] },
+    {"name":"stickers","parameterTypes":[] }
+  ]
 },
 {
   "name":"org.asamk.signal.manager.storage.stickers.LegacyStickerStore$Storage$Sticker",
   "allDeclaredFields":true,
   "queryAllDeclaredMethods":true,
   "queryAllDeclaredConstructors":true,
-  "methods":[{"name":"<init>","parameterTypes":["java.lang.String","java.lang.String","boolean"] }]
+  "methods":[
+    {"name":"<init>","parameterTypes":["java.lang.String","java.lang.String","boolean"] },
+    {"name":"packId","parameterTypes":[] },
+    {"name":"packKey","parameterTypes":[] },
+    {"name":"installed","parameterTypes":[] }
+  ]
 },
 {
   "name":"org.asamk.signal.util.SecurityProvider$DefaultRandom",

